snow/consensus/snowball/consensus_performance_test.go

```diff
@@ -29,10 +29,10 @@ func TestDualAlphaOptimization(t *testing.T) {
 		source        = prng.NewMT19937()
 	)
 
-	singleAlphaNetwork := NewNetwork(params, numColors, source)
+	singleAlphaNetwork := NewNetwork(SnowballFactory{}, params, numColors, source)
```

Comment:
I think this is purely stylistic. But I kind of like defining `SnowballFactory` as a global like we did with `trace.NoOp` [here](https://github.com/ava-labs/avalanchego/blob/master/trace/noop.go#L14-L16).

Reply:
Oh that looks way nicer, thanks 🙏